mpas_analysis/__main__.py

```diff
@@ -145,6 +145,9 @@ def build_analysis_list(config, controlConfig):
     analyses.append(ocean.ClimatologyMapEKE(config,
                                             oceanClimatolgyTasks['avg'],
                                             controlConfig))
+    analyses.append(ocean.ClimatologyMapBSF(config,
+                                            oceanClimatolgyTasks['avg'],
```

Comment:
```suggestion
                                            oceanClimatologyTasks['avg'],
```
I fixed this typo in #913 so it should be fixed here, too, once that gets merged and you rebase onto `develop`.

Reply:
@anirban89, this needs to be fixed now.  As soon as it is fixed, I'll re-test the analysis.

Reply:
I pushed a commit right now

Reply:
Perfect, thanks!